script/install.sh

```diff
@@ -371,13 +388,17 @@ Run \`deactivate\` later on to restore your environment."
 }
 
 install_dlang_installer() {
-    local url tmp
-    url=https://dlang.org/install.sh
+    local tmp mirrors
     tmp=$(mkdtemp)
+    mirrors=(
+        "https://dlang.org/install.sh"
+        "https://downloads.dlang.org/other/install.sh"
+        "https://nightlies.dlang.org/install.sh"
+    )
 
     mkdir -p "$ROOT"
-    log "Downloading $url"
-    download "$url" "$tmp/install.sh" "$url.sig"
+    log "Downloading ${mirrors[0]}"
+    download "${mirrors[*]}" "$tmp/install.sh" "${mirrors[*]/%/.sig}"
```

Comment:
Again `.sig` is implicit when `verify=1`, so the call would be.
```sh
download "$tmp/install.sh" 1 "${mirrors[@]}"
```
Or for better readability, use `download_verify` and plain `download`.